### PR TITLE
Annotation prop declaration refactor & `oio` refactor

### PR DESCRIPTION
**Pull request by joeflack4**
Annotation prop declaration refactor & oio refactor
    - Update: Annotation prop declaration UriRefs to use codebase standard (namespaces) instead of plain string URIs.
    - Delete: Duplicate oboInOwl namespace declaration
    - Update: All previous oboInOwl namespace usages to use the all-caps OBOINOWL, which is the standard for the codebase.

### CODE REVIEW

File: omim2obo/main.py
Comment (joeflack4) [inline omim2obo/main.py:1]: Reminder for when we review:

There are some bugs in GitHub's diff. I pushed an update. It says I'm deleting `OBOINOWL` and replacing it with `oboInOWL` in the diff, but there never was an all caps `OBOINOWL` before this PR.

File: omim2obo/main.py [hunk 1]
```diff
@@ -139,8 +139,9 @@ def omim2obo(use_cache: bool = False):
     # Populate graph
     # - Non-OMIM triples
     graph.add((URIRef('http://purl.obolibrary.org/obo/mondo/omim.owl'), RDF.type, OWL.Ontology))
-    graph.add((URIRef('http://www.geneontology.org/formats/oboInOwl#hasSynonymType'), RDF.type, OWL.AnnotationProperty))
-    graph.add((URIRef('http://purl.obolibrary.org/obo/mondo#omim_included'), RDF.type, OWL.AnnotationProperty))
+    graph.add((URIRef(oboInOwl.hasSynonymType), RDF.type, OWL.AnnotationProperty))
+    graph.add((URIRef(MONDONS.omim_included), RDF.type, OWL.AnnotationProperty))
+    graph.add((URIRef(MONDONS.ABBREVIATION), RDF.type, OWL.AnnotationProperty))
```
Comment (twhetzel) [inline omim2obo/main.py:144]: update to OMO abbreviation property

**Review by twhetzel — CHANGES_REQUESTED** [review]
comment inline

File: omim2obo/main.py (hunk 1 above)
Comment (joeflack4) [reply omim2obo/main.py:144]: Sounds good! I'll make that same change here.

Comment (joeflack4) [reply omim2obo/main.py:144]: You know, this was my solution before to our recent problems with pronto's error.. I would also declare that these are sub-properties of `oboInOwl:hasSynonymType`. It wouldn't hurt to do that here as well, either in this PR or later.

Comment (twhetzel) [reply omim2obo/main.py]: sure, whichever PR is best given how all these PRs/branches are being managed.

**Review by twhetzel — APPROVED** [review]
lgtm

**Pull request merged**